content/guide/ui/styling.md

```diff
@@ -0,0 +1,694 @@
+---
+title: Styling
+---
+
+In NativeScript you style the UI similarly to how you do it in a web application using Cascading Style Sheets (CSS), or by changing the style property of the views in JavaScript/Typescript. For CSS styling, NativeScript supports tools such as [Tailwind](https://docs.nativescript.org/plugins/tailwindcss.html), [SASS](#using-sass). 
```

Comment:
```suggestion
In NativeScript you style the UI similarly to how you do it in a web application using Cascading Style Sheets (CSS), or by changing the style property of the views in JavaScript/Typescript. For CSS styling, NativeScript supports tools such as [Tailwind](https://docs.nativescript.org/plugins/tailwindcss.html), [SASS](#using-sass). 
```

In NativeScript, UIs are styled similarly to how web applications using Cascading Style Sheets (CSS) are styled or by changing the style property of the views in JavaScript/TypeScript. For CSS styling, NativeScript supports tools such as [Tailwind](https://docs.nativescript.org/plugins/tailwindcss.html) and [SASS](#using-sass).

Reply:
@NathanWalker